scripts/convertConfig.ts

```diff
@@ -14,8 +14,8 @@ type HTML = string;
 type BookCollectionAudio = {
     num: number;
     src: string;
-    len: number;
-    size: number;
+    len?: number;
```

Comment:
If I have these set as optional, then the values in the JSON are `null` (to me indicates missing) as apposed to `0` (to me indicates zero length).

Reply:
Undefined not null but yes